docs/faq/add_op_in_backend.md

```diff
@@ -567,29 +568,66 @@ def test_quadratic_function():
     a = np.random.random_sample()
     b = np.random.random_sample()
     c = np.random.random_sample()
-    for ndim in range(1, 6):
```

Comment:
Why remove imperative test? Writing imperative test is the most intuitively straightforward for beginners. Suggest keep it and add symbolic test on the side.

Reply:
i have added.